mailchimp3/__init__.py

```diff
@@ -4,6 +4,7 @@
 
 Documentation at http://developer.mailchimp.com/documentation/mailchimp/reference/overview/
 """
+from mailchimp3.entities.customerjourney import CustomerJourney
```

Comment:
Formatting: This import needs to go right above the comment about E-Commerce and needs a comment above it about Customer Journeys. The imports here are listed in the order the API Documentation lists them and we would like to maintain that.